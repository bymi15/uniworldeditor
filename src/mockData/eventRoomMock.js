const mockEventRoom = {
  title: 'mockTitle',
  eventDate: 'mockDate',
  meetingTables: [],
  background: 'mockBackground',
  scene: 'mockScene',
};

const mockEventRoomUpdate = {
  title: 'newMockTitle',
  eventDate: 'newMockDate',
  meetingTables: [],
  background: 'newMockBackground',
  scene: 'newMockScene',
};

const mockMeetingTable = {
  title: 'mockTitle',
  type: 'RoundMeetingTable',
  logoUrl: 'https://via.placeholder.com/150',
  zoomUrl: 'https://zoom.us/',
  posX: 0,
  posY: 0,
};

export { mockEventRoom, mockEventRoomUpdate, mockMeetingTable };
