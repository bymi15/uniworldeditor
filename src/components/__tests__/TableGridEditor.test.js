import React from 'react';
import { shallow } from 'enzyme';
import TableGridEditor from '../TableGridEditor';
import { mockMeetingTable } from '../../mocks/eventRoomMock';

describe('TableGridEditor test', () => {
  it('TableGridEditor should match snapshot', () => {
    const component = shallow(
      <TableGridEditor
        tables={[mockMeetingTable]}
        updateMeetingTables={() => {}}
      />
    );
    expect(component).toMatchSnapshot();
  });
  it('TableGridEditor should match snapshot with empty tables array', () => {
    const component = shallow(
      <TableGridEditor tables={[]} updateMeetingTables={() => {}} />
    );
    expect(component).toMatchSnapshot();
  });
});
