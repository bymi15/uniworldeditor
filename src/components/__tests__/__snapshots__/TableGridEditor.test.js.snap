// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableGridEditor test TableGridEditor should match snapshot 1`] = `
<Fragment>
  <TableDialog
    onClose={[Function]}
    onSubmit={[Function]}
    open={false}
    updateTable={
      Object {
        "logoUrl": "https://via.placeholder.com/150",
        "posX": 0,
        "posY": 0,
        "title": "mockTitle",
        "type": "RoundMeetingTable",
        "zoomUrl": "https://zoom.us/",
      }
    }
  />
  <TableDialog
    onClose={[Function]}
    onSubmit={[Function]}
    open={false}
    updateTable={
      Object {
        "logoUrl": "https://via.placeholder.com/150",
        "posX": 0,
        "posY": 0,
        "title": "mockTitle",
        "type": "RoundMeetingTable",
        "zoomUrl": "https://zoom.us/",
      }
    }
  />
  <div
    className="makeStyles-gridContainer-1"
  >
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
      <WithStyles(ForwardRef(Avatar))
        src="roundtable.png"
      />
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
  </div>
</Fragment>
`;

exports[`TableGridEditor test TableGridEditor should match snapshot with empty tables array 1`] = `
<Fragment>
  <TableDialog
    onClose={[Function]}
    onSubmit={[Function]}
    open={false}
    updateTable={
      Object {
        "logoUrl": "https://via.placeholder.com/150",
        "posX": 0,
        "posY": 0,
        "title": "mockTitle",
        "type": "RoundMeetingTable",
        "zoomUrl": "https://zoom.us/",
      }
    }
  />
  <TableDialog
    onClose={[Function]}
    onSubmit={[Function]}
    open={false}
    updateTable={
      Object {
        "logoUrl": "https://via.placeholder.com/150",
        "posX": 0,
        "posY": 0,
        "title": "mockTitle",
        "type": "RoundMeetingTable",
        "zoomUrl": "https://zoom.us/",
      }
    }
  />
  <div
    className="makeStyles-gridContainer-1"
  >
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-gridItem-2 makeStyles-gridItemFill-3"
      onClick={[Function]}
    >
       
    </WithStyles(ForwardRef(Button))>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
    <div
      className="makeStyles-gridItem-2"
    >
       
    </div>
  </div>
</Fragment>
`;
